system/jlib/jsocket.cpp

```diff
@@ -1469,10 +1469,25 @@ void CSocket::setTraceName(const char * prefix, const char * name)
 #ifdef _TRACE
     StringBuffer peer;
     peer.append(prefix);
+    try
+    {
+        int lport;
+        char lname[256];
+        lport = CSocket::name(lname, sizeof(lname));
+        peer.append(":").append(lport);
+    }
+    catch (IException *e)
+    {
+        e->Release();
+    }
+    peer.append(" -> ");
     peer.append(name?name:"(NULL)");
+    peer.append(":").append(hostport);
+    if (sock != INVALID_SOCKET)
+        peer.append(" (").append(sock).append(")");
```

Comment:
lport is local port, hostport is remote port, good to see both and sock fd in trace.